mobile/src/reducers/settings.js

```diff
@@ -22,7 +22,7 @@ export const settings = (state = initialState, action) => {
     case INIT_STATE:
       return initialState
     case SET_CLIENT:
-      return { ...state, client: action.client, authorized: true }
+      return { ...state, authorized: true }
```

Comment:
I did not say to remove it. I said that the name was not reflecting the behavior anymore... Now it's in the code base, good luck for the 6 months old ourselves if they want to understand what does the code do.